.github/workflows/release.yml

```
@@ -33,6 +33,8 @@ jobs:
           helm repo add it-at-m https://it-at-m.github.io/helm-charts
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1.7.0
+        with: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)